### PR TITLE
Delete `credit-card` sample from paymentrequest

### PR DESCRIPTION
**Pull request by maxlgu**
Since the `basic-card` method of the Payment Request API has been deprecated, this PR deletes the `credit-card` sample.

Bug: [crbug.com/1299748](crbug.com/1299748)

### CODE REVIEW

**Comment by maxlgu** [comment]
@rsolomakhin , PTAL.

Other samples are using basic-card as well. We should remove them in separate PRs.

File: paymentrequest/payment-handler/index.html [hunk 1]
```diff
@@ -1,18 +1,17 @@
...
+feature_name: Payment Handler API
+chrome_version: 68
+feature_id: 5160285237149696
 ---
 
 <h3>Background</h3>
 <p>
   <a href="https://github.com/w3c/browser-payment-api">PaymentRequest</a> lets
-  you accept payment from different payment methods.
+  you accept payment with a URL-based payment method.
```
Comment (rsolomakhin) [inline paymentrequest/payment-handler/index.html]: We still support non-URL payment methods such as "secure-payment-confirmation", so please leave the original text as it was here.

File: paymentrequest/payment-handler/index.html [hunk 2]
```diff
@@ -1,18 +1,17 @@
...
 </p>
 
 <p>
-  This sample accepts credit card payments and does not request shipping
-  information.
+  This sample accepts the a URL-based payment method.
```
Comment (rsolomakhin) [inline paymentrequest/payment-handler/index.html]:
```suggestion
  This sample accepts a URL-based payment method.
```

**Review by rsolomakhin — CHANGES_REQUESTED** [review]

**Comment by maxlgu** [comment]
Updated the text. PTAL.

File: paymentrequest/payment-handler/index.html [hunk 3]
```diff
@@ -7,11 +7,11 @@
 <h3>Background</h3>
 <p>
   <a href="https://github.com/w3c/browser-payment-api">PaymentRequest</a> lets
-  you accept payment with a URL-based payment method.
+  you accept payment from different payment methods.
 </p>
 
 <p>
-  This sample accepts the a URL-based payment method.
+  This sample accepts the the URL-based payment method.
```
Comment (rsolomakhin) [inline paymentrequest/payment-handler/index.html]: Please remove the duplicate "the"

Comment (maxlgu) [reply paymentrequest/payment-handler/index.html]: Ack. It has been fixed in the last commit.

**Comment by maxlgu** [comment]
+@jonathanGB who may have write access, could you review this? Thanks!

**Comment by maxlgu** [comment]
@rsolomakhin , do you have the write access? could you approve it? thanks!

**Comment by rsolomakhin** [comment]
Thank you, @jeffposnick!

**Pull request merged**